tests/phpunit/Admin/WC_REST_Stripe_Settings_Controller_GB_Test.php

```diff
@@ -60,7 +57,7 @@ public static function set_up_before_class() {
 		$upe_helper->enable_upe();
 		$upe_helper->reload_payment_gateways();
 
-		self::$gateway = WC()->payment_gateways()->payment_gateways()[ WC_Gateway_Stripe::ID ];
+		self::$gateway = WC()->payment_gateways()->payment_gateways()[ WC_Stripe_UPE_Payment_Gateway::ID ];
```

Comment:
Not blocking: we still have plenty of references to `WC_Gateway_Stripe::ID` in the main code. But we can tackle them in other PRs.